src/main/java/com/fnp/checkexception/unchecked-exceptions-program/UncheckedException.java

```diff
@@ -0,0 +1,17 @@
+import customexception.*;
+
+class UncheckedException{	
+	public static void main(String [] args){
+	
+	try{
+			ClassWhichThrowsException obj = new ClassWhichThrowsException();
+			obj.conditionChecker(0);
+	}
+			
+	catch(ObservedException e){
```

Comment:
Do we really need to catch a checked exception ??

Reply:
we have to catch a checked exception but not the unchecked exception.

Reply:
yes that's what I am also saying we should not catch it.